e2e/tests/transfer/forwarding_test.go

```diff
@@ -95,3 +99,69 @@ func (s *TransferForwardingTestSuite) TestThreeChainSetup() {
 		s.Require().Equal(expected, actualBalance.Int64())
 	})
 }
+
+func (s *TransferForwardingTestSuite) TestForwarding_WithLastChainBeingICS20v1_Succeeds() {
+	ctx := context.TODO()
+	t := s.T()
+
+	relayer, chains := s.GetRelayer(), s.GetAllChains()
+
+	chainA, chainB, chainC := chains[0], chains[1], chains[2]
+
+	channelAtoB := s.GetChainAChannel()
+
+	// Creating a new path between chain B and chain C with a ICS20-v1 channel
+	opts := s.TransferChannelOptions()
+	opts.Version = transfertypes.V1
+	channelBtoC, _ := s.CreateNewPath(ctx, chainB, chainC, ibc.DefaultClientOpts(), opts)
+	s.Require().Equal(transfertypes.V1, channelBtoC.Version, "the channel version is not ics20-1")
+
+	chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+	chainAAddress := chainAWallet.FormattedAddress()
+	chainADenom := chainA.Config().Denom
+
+	chainCWallet := s.CreateUserOnChainC(ctx, testvalues.StartingTokenAmount)
+	chainCAddress := chainCWallet.FormattedAddress()
+
+	t.Run("IBC transfer from A to C with forwarding through B", func(t *testing.T) {
+		inFiveMinutes := time.Now().Add(5 * time.Minute).UnixNano()
```

Comment:
[q] can we use here https://github.com/cosmos/ibc-go/blob/feat%2Fics20-v2-path-forwarding/testing/chain.go#L634 instead?

Reply:
Unfortunately no. I have not found a nice way to do this. The best option I had was a function that queries the latest header and gets the current block time from that. But since these are anyway running in real time in the background, I think just using current time will always work. 

Two options I see to improve this:
- Implement the query of latest block header or status from a node
- Make a little helper function for getting a reasonable timeout (just putting this in a function, basically)